provider/provider_test.go

```diff
@@ -49,11 +49,14 @@ func testProviderUpgradeWithConfig(t *testing.T, dir, baselineVersion string, co
 		opttest.LocalProviderPath(providerName, filepath.Join(cwd, "..", "bin")),
 	)
 	test.SetConfig("gcp:config:project", testProject)
+	test.SetConfig("gcp:config:skipRegionValidation", "true")
 	if config != nil {
 		for k, v := range config {
 			test.SetConfig(k, v)
 		}
 	}
+	// The SetConfig above does not seem to be working here.
+	test.T().Setenv("PULUMI_GCP_SKIP_REGION_VALIDATION", "true")
```

Comment:
I think this is https://github.com/pulumi/providertest/issues/56 will look into it later.